test/main/xdShare.t.sol

```diff
@@ -230,6 +225,77 @@ contract xdShareTest is Test {
         }
     }
 
+    function testDepositYieldRebaseYieldRedeem() public {
+        uint128 amount = 1000;
+        uint128 balancePerShare = 42 ether;
+        // vm.assume(amount > 0);
```

Comment:
if we don't fuzz, why keep commenting on fuzzy logic

Reply:
fixed